packages/@lwc/engine-server/src/__tests__/fixtures/light-dom-styles-dir-host/expected.html

```diff
@@ -0,0 +1,11 @@
+<x-basic class="lwc-1rssj1tib70-host">
+  <style class="lwc-1rssj1tib70" type="text/css">
+     :dir(rtl) {margin-left: 0;}:host {color: red;}
+  </style>
+  <style class="lwc-1rssj1tib70" type="text/css">
+     :dir(ltr) {margin-right: 0;}.lwc-1rssj1tib70-host {color: blue;}
+  </style>
```

Comment:
It's always `:dir()` because we don't support synthetic shadow, but light DOM gets the "fake" `:host()` when using scoped styles only.